quickstart/src/main/java/com/datastax/oss/quarkus/FruitDaoReactive.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.quarkus;
+
+import com.datastax.oss.driver.api.mapper.annotations.Dao;
+import com.datastax.oss.driver.api.mapper.annotations.Select;
+import com.datastax.oss.driver.api.mapper.annotations.Update;
+import com.datastax.oss.quarkus.runtime.api.reactive.MutinyMappedReactiveResultSet;
+import java.util.concurrent.CompletionStage;
+
+@Dao
+public interface FruitDaoReactive {
+
+  @Update
+  CompletionStage<Void> updateAsync(Fruit fruitDao);
```

Comment:
This is not reactive, why not return a `Mono` or whatever it's called?

Reply:
The mutiny api integration works well with `CompletionStage` so I think we are safe to use it. Regarding returning `Mono` we would need to adapt the mapper to return it, so I am not sure if this is worth the effort.

Reply:
After more investigation, it seems that it is a good idea to convert it to `Uni` (Mongodb for the `ReactiveMongoClient` is exposing `Uni`) so I'll do.